### ISSUE TITLE
Kotlin: Package level functions cannot be imported from library project with new gradle plugin

### ISSUE THREAD

**Issue by wmbest2** [realm/realm-java#2356]
This is a pretty elaborate case so hopefully I can explain it well enough.

We have a project consisting of a main application and a library project.  The library project contains Realm models that are shared.  Using the new plugin on both the main application and the library project causes the main project to not see the package level functions in the library.

I've attached a small sample project to show what I mean.

Given this file in the library project 

~~~kotlin
package io.vokal.realmtest.model

import io.realm.*
import io.realm.annotations.*

@RealmClass
open class Dog(
    @PrimaryKey
    public open var name: String = "spot"
): RealmObject() { }

@RealmModule(library = true, allClasses = true)
public class LibraryModule {
}

public fun Realm.findSpot(): Dog {
    return where(Dog::class.java).equalTo("name", "spot").findFirst()
}
~~~

I should be able to import `findSpot` in the main activity.  

Instead I receive `e: /home/wmbest2/realm-library-test/app/src/main/kotlin/io/vokal/realmtest/MainActivity.kt: (37, 23): Unresolved reference: findSpot`

[realm-library-test.zip](https://github.com/realm/realm-java/files/149012/realm-library-test.zip)

**Comment by zaki50** [comment]
@wmbest2 Thank you for reporting.

I'll look into attached project.

**Comment by zaki50** [comment]
I confirmed that our bytecode transformer triggers this issue.

**Comment by zaki50** [comment]
I found a difference.

`META-INF` disappeared when our bytecode transformer is enabled.

![2016-02-27 23 57 11](https://cloud.githubusercontent.com/assets/406660/13373546/f20b0b02-ddad-11e5-9583-cbf936b1458d.png)

**Comment by wmbest2** [comment]
Awesome! I figured it was something related to the metadata that the kotlin compiler looks for. Thanks for quick response.

**Comment by zaki50** [comment]
I fixed this issue on my local environment.
I'll make a PR in a few days(I need some more checks and also need to write the unit test).

Sorry for the inconvenience.

**Comment by zaki50** [comment]
I used following project. That's almost same with one attached to first comment but modified to use local Realm build instead of SNAPSHOT release.

[realm-library-test_issue2356.zip](https://github.com/realm/realm-java/files/149459/realm-library-test_issue2356.zip)

**Comment by wmbest2** [comment]
@zaki50 looks like you might have forgotten to include the libs folder.  Unless that was your intention in which case I'll wait for the new snapshot to verify.

**Comment by zaki50** [comment]
@wmbest2 sorry for late reply. I caught a cold but now I get over it.

I'll test the libs folder.

**Comment by zaki50** [comment]
@wmbest2 I merged #2358 into master.

**Comment by wmbest2** [comment]
Thank you! I've verified that it is now working as expected
